app-editors/qhexedit2/metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>coyote@bks.tv</email>
+        <name>Victor Kustov</name>
+        <description>Proxied-maintainer. Assign bugs to him</description>
+    </maintainer>
+    <maintainer type="person">
+        <email>zlogene@gentoo.org</email>
+        <name>Mikle Kolyada</name>
+    </maintainer>
+    <maintainer type="person">
+        <email>maksbotan@gentoo.org</email>
+        <name>Maxim Koltsov</name>
+    </maintainer>
+    <maintainer type="project">
+        <email>proxy-maint@gentoo.org</email>
+        <name>Proxy Maintainers</name>
+    </maintainer>
+    <use>
+        <flag name="doc">Install documentation</flag>
+        <flag name="example">Build and install example - Qt5 hex editor</flag>
```

Comment:
`examples` is the global word. Does any source code get installed, too? If not, then maybe use gui.

Reply:
gui is better, you're right.